docs/build_and_load.md

```diff
@@ -1198,6 +1204,10 @@ In addition, `--conda-env-prefix` is an optional parameter that specifies the pa
 to the Conda directory that contains the conda environment needed to run phg.
 If not set, conda env `phgv2-conda` in the defauilt location will be used.
 
+There is also an optional parameter: `--legacy-maf-file`.  This flag can be added to the command
+to be able to process MAF files created from anchorwave 1.2.2 or earlier.  If this is not set and 
```

Comment:
I've been thinking users won't know they phg version, but they will if they know what is in their phg_environment.yml file.  Or if they ran "anchorwave --version".  

Should we tell them explicitly how to check their anchorwave version?  Let them know that merely downloading the new PHG does not update their conda environment, but that they need to run commands to explicitly update it?

Reply:
I think "how to check the version of software I have installed" is general enough information that we don't need to include it. Especially since we're using conda and any search to that effect returns `conda list` as a viable option. But I agree that reminding them they need to update the conda environment manually can't hurt.